kb/deadline-exceeded-troubleshooting.md

```diff
@@ -110,4 +110,4 @@ If you increase `frontend.keepAliveMaxConnectionAge` values, consider monitoring
 
 ---
 
-If you were unable to resolve your issue, check for similar questions and possible solutions on our [community forum](https://community.temporal.io) or [community Slack](https://temporal.io/slack), or file a [support ticket](https://support.temporal.io/).
+If you were unable to resolve your issue, check for similar questions and possible solutions on our [community forum](https://community.temporal.io) or [community Slack](https://temporal.io/slack), or file a [support ticket](https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support).
```

Comment:
If this is specific to cloud, i'd put open support ticket first before recommending community resources to customers, but of course they can check those too